resources/views/equipment/repairs/view.blade.php

```diff
@@ -56,6 +56,26 @@
             </div>
         </div>
 
+        <div class="form-group">
+            {!! Form::label('images', 'Images:', ['class' => 'control-label col-md-4']) !!}
+            <div class="col-md-8">
+                @if($breakage->images->count())
+                    <div class="row">
+                        @foreach($breakage->images as $image)
+                            <div class="col-xs-6 col-md-4">
+                                <a href="{{ $image->getImageRoute() }}" class="thumbnail" target="_blank">
+                                    <img src="{{ $image->getImageRoute() }}" alt="Image">
+                                </a>
+                            </div>
+                        @endforeach
+                    </div>
+                @else
+                    <p class="form-control-static"><em>- no images -</em></p>
+                @endif
+            </div>
+
+        </div>
```

Comment:
We should probably wrap this entire form group in the if statement, so that we only show the entire thing if there are images. Also, I'd prefer a more explicit check so we don't have type coercion:

```blade
@if($breakage->images->count() > 0)
    ...
@endif
```

You could also move that to a helper function on the breakage if you wanted (eg, `hasImages`) but that's probably overkill.

Reply:
Hmm, actually I'm less sure about this - definitely want the stricter check though!